client-libs/typescript/tsup.config.ts

```diff
@@ -0,0 +1,25 @@
+import { Options } from "tsup";
+
+const config: Options = {
+  splitting: false, // Disable code splitting
+  sourcemap: true, // Include sourcemaps
+  target: "es2020", // Target ES2020 syntax for modern environments
+  dts: true, // Generate declaration files
+
+  // Define entry points
+  entry: [
+    "src/index.ts", // Main entry
+    "src/openai.ts", // 'openai' sub-module
+    "src/openai/index.test.ts", // 'openai' sub-module
```

Comment:
Why does this need to be an entrypoint? Don't we just need src/index and src/openai?

Reply:
Yes, you are correct, I thought I had removed that.